sdc-nfs-config.json

```diff
@@ -0,0 +1,19 @@
+{
+    "portmap": {
+        "usehost": 1
```

Comment:
From https://github.com/joyent/sdc-nfs:

> The server includes a built-in portmapper, but it will also interoperate transparently with the zone's portmapper (usually `rpcbind`) if one is running. The server also includes a built-in `mountd` and `nfsd`. There is no `lockd` provided by the server.

...but I get errors when setting this to `0`